scripts/api_review/README.md

```diff
@@ -1,3 +1,9 @@
 # Elektra API Review Template Script
 
 This python script generates new review files from the given template.
+
+The script takes a list of methods as well as a template as input parameters.
+The list of methods should be a file where each line contains a signature of a function that should be reviewed. A review file will be generated for each line.
+The template gets passed two variables: signature and name of the function which can be accessed in the template via `{signature}` and `{name}` respectively.
```

Comment:
```suggestion
The template gets passed two variables:
signature and name of the function which can be accessed in the template via `{signature}` and `{name}` respectively.
```

Reply:
Technically, this is one sentence, even if there is a colon, since the part after the colon is not a full sentence (has no verb/predicate). You could also write it like this to make it more clear that it's just one sentence:

```suggestion
The template gets passed two variables, the signature and name of the function, which can be accessed in the template via `{signature}` and `{name}` respectively.
```